types/netmap/netmap.go

```diff
@@ -66,10 +76,6 @@ type NetworkMap struct {
 	// hash of the latest update message to tick through TKA).
 	TKAHead tka.AUMHash
 
-	// ACLs
-
-	User tailcfg.UserID
```

Comment:
would this break GUIs?

Reply:
It broke Windows but I adjusted it in https://github.com/tailscale/corp/pull/14065

Reply:
All xcode builds seem happy locally.